README.md

```diff
@@ -102,10 +102,12 @@ By default it will use `${HOME}/notes` as your notes directory and if that
 directory doesn't exist beforehand, this script will allow you to create it
 with a `y/n` prompt when you first run the program.
 
-You can also customize your notes path in 1 of 2 ways:
+You can also customize your notes path in 1 of 3 ways:
```

Comment:
Can you replace all of your changes and completely update the `### Configuration` documentation to be this:

```md
### Configuration

You can configure where notes get saved to (`NOTES_DIRECTORY`) along with the
file extension (`NOTES_EXTENSION`).

#### `NOTES_DIRECTORY`

By default it will use `${HOME}/notes` as your notes directory and if that
directory doesn't exist beforehand this script will allow you to create it
with a `y/n` prompt when you first run the program.

For example you could set `NOTES_DIRECTORY=/tmp` to save your notes there
instead.

#### `NOTES_EXTENSION`

By default it will use `txt` as the file extension but if you want to use
something else such as Markdown you can set this to `NOTES_EXTENSION=md`.

#### Customizing any of the variables

You can configure any options by going to your `~/.profile` or equivalent file
and then adding this line `export NOTES_EXTENSION=md` or whatever you want to
customize.

*If you change your `~/.profile`, don't forget to log out / login.*

#### Automatically opening your default editor

If you want this script to open your notes in your preferred code editor you'll
want to make sure you have your `EDITOR` defined in your `~/.profile` too. This
is a Unix standard. For example, mine looks like `export EDITOR="vim"`.
```